ansible/roles/provision-vm/tasks/main.yml

```diff
@@ -36,7 +36,7 @@
 # At this point, ansible should be able to run on the remote host
 
 - wait_for_connection:
-    timeout: 180
+    timeout: 600
```

Comment:
Can it really take 10 minutes to reconnect at this point?

Reply:
changed to default 180